system/Database/Forge.php

```diff
@@ -541,10 +545,10 @@ protected function _createTable(string $table, bool $ifNotExists, array $attribu
         $columns = implode(',', $columns);
 
         $columns .= $this->_processPrimaryKeys($table);
```

Comment:
I changed _processIndexes() to return an array rather than a string. current() is used to get the first (and only) element of array.